vendor/modules.txt

```diff
@@ -8,13 +8,13 @@ github.com/BurntSushi/toml
 # github.com/VividCortex/mysqlerr v0.0.0-20170204212430-6c6b55f8796f
 ## explicit
 github.com/VividCortex/mysqlerr
-# github.com/apache/arrow/go/arrow v0.0.0-20191025121910-b789226ccb21
+# github.com/apache/arrow/go/arrow v0.0.0-20200403134915-89ce1cadb678
```

Comment:
Probably something to think about in SDK and use a proper tagged version of arrow?

Reply:
Apparently, as of 1 hour ago https://twitter.com/ApacheArrow/status/1253347469313953793